include/caf/sensor_sampler.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _SENSOR_SAMPLER_H_
+#define _SENSOR_SAMPLER_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <zephyr/types.h>
+
+struct sampled_channel {
+	enum sensor_channel chan;
+	uint8_t data_cnt;
+};
+
+struct sensor_config {
+	const char *dev_name;
+	const char *event_descr;
```

Comment:
Do we need `*event_descr` at all? Couldn't the sensor be recognized by its name? Furthermore does `event_descr` belong to `sensor_config`? Those should be separate things.

Reply:
`event_descr` is used to identify both sensor and the selected channels (we may sample only subset of available channels).

It allows application to identify events and it should be configurable by user. It's why it is added as part of module configuration.

Reply:
Okay, I understand.